### PR TITLE
Add e2e tutorial for fraud detection

### PR DESCRIPTION
**Pull request by lucferbux**
Closes https://issues.redhat.com/browse/RHOAIENG-532

## Description

Adding a dashboard tile for the "OpenShift AI tutorial - Fraud detection example" guide.

## How Has This Been Tested?

1. Deploy all the rhoai documentation with `oc apply -k manifests/overlays/apps/base/rhoai`
2. Check that there's a new doc in the resources section

Screenshot 2024-03-01 at 12 23 47

## Test Impact

No testing required, new resource added, no changes in the codebase

## Request review criteria:

Self checklist (all need to be checked):
- [X] The developer has manually tested the changes and verified that the changes work
- [X] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Review by manaswinidas — COMMENTED** [review]
/lgtm

Tested. Works fine. I can see the new docs with logo and appName

**Review by dpanshug — APPROVED** [review]
/lgtm

**Comment by lucferbux** [comment]
Gonna approve to fix the issue that will affect other PRs

File: manifests/overlays/apps/base/rhoai/rhoai-docs.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  name: rhoai-tutorial-fraud
+  annotations:
+    opendatahub.io/categories: 'Getting started,Model training,Notebook environments,Pipelines'
+spec:
+  displayName: OpenShift AI tutorial - Fraud detection example
+  appName: rhoai
+  type: how-to
+  description: |-
+    Use OpenShift AI to develop and train an example model in Jupyter Notebooks, deploy the model, integrate the model into a fraud detection application, and refine the model by using automated pipelines.
```
Comment (MelissaFlinn) [inline manifests/overlays/apps/base/rhoai/rhoai-docs.yaml:12]: Can we change the description to:
Use OpenShift AI to train an example model in a Jupyter notebook, deploy the model, integrate the model into a fraud detection application, and refine the model by using automated pipelines

**Pull request merged**